lib/auth/usertoken.go

```diff
@@ -148,8 +148,13 @@ func (a *Server) CreateResetPasswordToken(ctx context.Context, req CreateUserTok
 		return nil, trace.BadParameter("invalid reset password token request type")
 	}
 
-	_, err = a.ResetPassword(ctx, req.Name)
-	if err != nil {
+	// Since ResetPassword ignores doesn't report errors for non-existent users,
+	// first check if the user exists.
```

Comment:
There are a few extra words in there, rephrase?

```suggestion
	// ResetPassword doesn't error for non-existent users, so first check if the
	// user exists.
```